metrics/mdl_high_water_mark

```diff
@@ -1 +1 @@
-3
+5
```

Comment:
There's a couple of long lines in code blocks in the new driver-writing document.  Unfortunately what I'm doing in the markdownlint config to try to allow that isn't working for some reason.  Asking for two-developer approval on this one.